### PR TITLE
Bump to xamarin/java.interop@cedf4d0

### PR DESCRIPTION
**Pull request by pjcollins**
* Bump to xamarin/java.interop@cedf4d0
* [Mono.Android] Do not check target framework of referenced console apps
These project references only exist to ensure a particular build order,
so we can skip target framework version compatibility checks.

### CODE REVIEW

File: src/Xamarin.Android.Build.Tasks/Xamarin.Android.Build.Tasks.targets [hunk 1]
```diff
@@ -322,6 +322,7 @@
       <_ILRepackArgs>/out:&quot;$(MSBuildThisFileDirectory)$(IntermediateOutputPath)$(AssemblyName).dll&quot; /internalize</_ILRepackArgs>
       <_ILRepackArgs>$(_ILRepackArgs) &quot;$(MSBuildThisFileDirectory)$(IntermediateOutputPath)$(AssemblyName).dll&quot;</_ILRepackArgs>
       <_ILRepackArgs>$(_ILRepackArgs) @(_InputAssembliesThatExist->'&quot;%(Identity)&quot;', ' ')</_ILRepackArgs>
+      <_ILRepackArgs>$(_ILRepackArgs) /lib:&quot;$(XAInstallPrefix)xbuild-frameworks\MonoAndroid\v1.0\facades&quot;</_ILRepackArgs> <!-- Path to netstandard.dll -->
```
Comment (jonathanpeppers) [inline src/Xamarin.Android.Build.Tasks/Xamarin.Android.Build.Tasks.targets]: I'm wondering if this is right... `netstandard.dll` in this case is coming from our build tree, and meant for Xamarin.Android projects. Xamarin.Android.Build.Tasks is a desktop assembly, should it be using Mono's system install of `netstandard.dll` instead?

**Comment by jonpryor** [comment]
Squash-and-merge commit message:

```
Changes: https://github.com/xamarin/java.interop/compare/bd7c60a6a5ea8abaf6f241da17c42d2d93226bf2...cedf4d04c7bf1f327d615a0316cbeabe59be5454

  * xamarin/Java.Interop@cedf4d0: [build] Convert more projects to Short-Form projects (#547)

Update `src/Mono.Android` to not check target framework of referenced
console apps.  These project references only exist to ensure a
particular build order, so we can skip target framework version
compatibility checks.
```

File: src/Xamarin.Android.Build.Tasks/Xamarin.Android.Build.Tasks.targets (hunk 1 above)
Comment (pjcollins) [reply src/Xamarin.Android.Build.Tasks/Xamarin.Android.Build.Tasks.targets]: We were previously repacking against the netstandard.dll that was copied into this directory by some Java.Interop project. Maybe we can do something similar to what macios is doing here... https://github.com/xamarin/xamarin-macios/blob/0557f16e6a936f9d3c94fe020536141296b42640/msbuild/ILMerge.targets#L13

Comment (pjcollins) [reply src/Xamarin.Android.Build.Tasks/Xamarin.Android.Build.Tasks.targets]: I just rebuilt on macOS with the latest changes, and we should now using a path from the .NET Core installation:

> /lib:"/usr/local/share/dotnet/sdk/NuGetFallbackFolder/netstandard.library/2.0.3/build/netstandard2.0/ref/"

Comment (pjcollins) [reply src/Xamarin.Android.Build.Tasks/Xamarin.Android.Build.Tasks.targets]: Looks like Windows wasn't happy with this approach

**Comment by pjcollins** [comment]
Re-running, we hit a bot which doesn't have powershell installed...
/azp run

**Pull request merged**